pkg/toolset/database.go

```diff
@@ -0,0 +1,258 @@
+package toolset
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	databasecore "github.com/gohornet/hornet/core/database"
+	"github.com/gohornet/hornet/pkg/database"
+	"github.com/gohornet/hornet/pkg/keymanager"
+	"github.com/gohornet/hornet/pkg/model/coordinator"
+	"github.com/gohornet/hornet/pkg/model/hornet"
+	"github.com/gohornet/hornet/pkg/model/milestone"
+	"github.com/gohornet/hornet/pkg/model/milestonemanager"
+	"github.com/gohornet/hornet/pkg/model/storage"
+	"github.com/gohornet/hornet/pkg/restapi"
+	"github.com/gohornet/hornet/pkg/snapshot"
+	"github.com/gohornet/hornet/pkg/utils"
+	"github.com/iotaledger/hive.go/serializer"
+	iotago "github.com/iotaledger/iota.go/v2"
+)
+
+const (
+	// the amount of public keys in a milestone.
+	CfgProtocolMilestonePublicKeyCount = "protocol.milestonePublicKeyCount"
+	// the ed25519 public key of the coordinator in hex representation.
+	CfgProtocolPublicKeyRanges = "protocol.publicKeyRanges"
+	// the ed25519 public key of the coordinator in hex representation.
+	CfgProtocolPublicKeyRangesJSON = "publicKeyRanges"
+	// subfolder for the tangle database
+	TangleDatabaseDirectoryName = "tangle"
```

Comment:
there are constants for these already

Reply:
I didn't want to import the "core plugins" here :thinking:

Reply:
Ah lel, I already imported them anyway.